src/home/home.scss

```diff
@@ -16,7 +16,7 @@
     &.fairLaunch {
       margin-left: 4.5em;
       margin-right: 4.5em;
-      margin-bottom: 5.75em;
+      margin-bottom: 8em;
```

Comment:
is there a px value we can use from the wireframe?

Reply:
same for the margins

Reply:
Maybe a PR to change some values to their px values because some component also uses this particular values. By the way do you want to remove all uses of em and rem or just for margins and paddings ?

Reply:
For now just do it when we're in the code and looking at the wireframe, like this code here.

Reply:
I think we should us px for everything, frankly